Document best practices for script and config params (#6631)
nextflow-io/nextflow: docs/tutorials/static-types.md

See {ref}`workflow-params-def` for more information about the `params` block.

:::{note}

> Parameters that are only used in the config file should be declared in the config file, not the script. Since rnaseq-nf does not have any such parameters, all parameters are declared in the script. See {ref}`config-params` for more information.

Reviewer: ```suggestion
Parameters used only in the config file should be declared there, not in the script. Since rnaseq-nf has no such parameters, all parameters are declared in the script. See {ref}`config-params` for more information.
```
